### DISCUSSION TITLE
Copilot works - sometimes?

### DISCUSSION THREAD

**Discussion by Riddle1495** [community/community#12431] [Copilot Conversations] [answered]
Started experiencing this bug a couple of days ago. I've restarted my VS code / PC since then and the bug is still happening.

It seems to be when the file I'm working on is > 100 lines.
I can create a new file and do type

// create an add function.

and it'll do it. But when do this in a bigger file it won't (it used to 3 days ago). Actually, it doesn't suggest anything if a file is > 100 lines (just a guess on how many lines).
I looked at the output for the copilot extension and I found this was being logged.

![image](https://user-images.githubusercontent.com/21700327/156866965-10b631b0-2cf9-45b3-ab87-6530bc5f0124.png)

**Comment by Riddle1495** [comment, marked as answer]
The error appears to be associated with the commented-out line in the picture below. Now that the code is commented out, it "works". I don't see any difference in the results from last week (when it was working correctly) and now.

![image](https://user-images.githubusercontent.com/21700327/156867364-cdcad588-9086-4ce4-93eb-0087fd0973ae.png)

**Reply by davecheney** [reply]
Thank you for this bug report. I'm following up internally.

**Reply by davecheney** [reply]
Hello. This problem should be fixed. Sorry for the disruption.

**Reply by Riddle1495** [reply]
Thanks.
